tests/profile/test_keyword.py

```diff
@@ -0,0 +1,22 @@
+"""
+This module contains the tests for the keyword profile.
+"""
+from unittest import mock
+
+from spokestack.profile.keyword import SpokestackKeyword
+
+
+@mock.patch("spokestack.profile.keyword.PyAudioInput")
+@mock.patch("spokestack.profile.keyword.KeywordRecognizer")
+@mock.patch("spokestack.profile.keyword.SpeechPipeline")
+def test_activate(*args):
+    pipeline = SpokestackKeyword.create(
+        classes=[
+            "one",
+            "two",
+            "three",
+        ],
+        model_dir="mock_model_dir",
+    )
+    pipeline.start()
+    pipeline.run()
```

Comment:
It would be good to get into the habit of also writing test cases for sad paths—in this case, invalid configurations.